sqlx-core/src/config/reference.toml

```diff
@@ -15,6 +15,12 @@
 # If not specified, defaults to `DATABASE_URL`
 database-url-var = "FOO_DATABASE_URL"
 
+[common.drivers.sqlite]
+# Load extensions into SQLite when running macros or migrations
+#
+# Defaults to an empty list, which has no effect.
+load_extensions = ["uuid", "vsv"]
```

Comment:
The parsing test should be extended to cover this: https://github.com/launchbadge/sqlx/blob/a0e40540e3496a53d6655c45b8c66626c7d74cc4/sqlx-core/src/config/tests.rs#L5